src/main/scala/ch/openolitor/buchhaltung/rechnungsexport/iso20022/Pain008_001_02_Export.scala

```diff
@@ -37,15 +37,26 @@ import scala.xml.{ NamespaceBinding, TopScope }
 
 class Pain008_001_02_Export extends LazyLogging {
 
+  val notProvided = "NOTPROVIDED"
+
   private def exportPain008_001_02(rechnungen: List[(Rechnung, KontoDaten)], kontoDatenProjekt: KontoDaten, NbOfTxs: String, projekt: Projekt): String = {
 
-    "<?xml version=\"1.0\" encoding=\"UTF-8\"?>" +
-      scalaxb.toXML[ch.openolitor.generated.xsd.pain008_001_02.Document](ch.openolitor.generated.xsd.pain008_001_02.Document(
-        CustomerDirectDebitInitiationV02(
-          getGroupHeaderSDD(rechnungen.map(_._1), kontoDatenProjekt, NbOfTxs, projekt),
-          Seq(getPaymentInstructionInformationSDD(projekt, kontoDatenProjekt, rechnungen, NbOfTxs))
-        )
-      ), "Document", defineNamespaceBinding()).toString()
+    NbOfTxs.length match {
+      case txs if txs <= 15 => {
+        getPaymentInstructionInformationSDD(projekt, kontoDatenProjekt, rechnungen, NbOfTxs) match {
+          case Some(pmtInf) =>
+            getGroupHeaderSDD(rechnungen.map(_._1), kontoDatenProjekt, NbOfTxs, projekt) match {
+              case Some(grpHeader) =>
+                "<?xml version=\"1.0\" encoding=\"UTF-8\"?>" +
+                  scalaxb.toXML[ch.openolitor.generated.xsd.pain008_001_02.Document](ch.openolitor.generated.xsd.pain008_001_02.Document(
+                    CustomerDirectDebitInitiationV02(grpHeader, Seq(pmtInf))
+                  ), "Document", defineNamespaceBinding()).toString()
+            }
+          case _ => ""
```

Comment:
As discussed: Maybe do some log output so we can debug more easily in case of failure.